### ISSUE TITLE
Compatibility of TAESD VAE Encoder Output with SD Original VAE

### ISSUE THREAD

**Issue by aidenyzhang** [madebyollin/taesd#25]
Thank you for your work on TAESD. I have a quick question about compatibility:

Can the VAE encoder output from TAE-SD be directly used with the decoder trained on SD-VAE-FT-MSE? Or is retraining the U-Net necessary?

Any clarification would be appreciated.

**Comment by madebyollin** [comment]
Yeah, the encoder outputs should be compatible (though not 100% identical), so any model that was trained with SD-VAE can also be used with TAESD. I posted an example [here](https://github.com/madebyollin/taesd/issues/12) showing that you can even encode an image with TAESD and decode with the SD-VAE.
